src/test/store/__snapshots__/profile-view.test.js.snap

```diff
@@ -1032,14 +1040,93 @@ Array [
 ]
 `;
 
+exports[`snapshots of selectors/profile matches the last stored run of markerThreadSelectors.getFullMarkerListIndexes 1`] = `
+Array [
+  Object {
+    "data": null,
+    "dur": 0,
+    "name": "A",
+    "start": 0,
+    "title": null,
+  },
+  Object {
+    "data": null,
+    "dur": 0,
+    "name": "B",
+    "start": 1,
+    "title": null,
+  },
+  Object {
+    "data": null,
+    "dur": 0,
+    "name": "C",
+    "start": 2,
+    "title": null,
+  },
+  Object {
+    "data": null,
+    "dur": 0,
+    "name": "D",
+    "start": 3,
+    "title": null,
+  },
+  Object {
+    "data": null,
+    "dur": 0,
+    "name": "E",
+    "start": 4,
+    "title": null,
+  },
+  Object {
+    "data": null,
+    "dur": 0,
+    "name": "F",
+    "start": 5,
+    "title": null,
+  },
+  Object {
+    "data": Object {
+      "URI": "https://mozilla.org",
+      "endTime": 7,
+      "fetchStart": 6.5,
+      "id": 6,
+      "pri": 0,
+      "startTime": 6,
+      "status": "STATUS_STOP",
+      "type": "Network",
+    },
+    "dur": 1,
+    "name": "Load 6: https://mozilla.org",
+    "start": 6,
+    "title": null,
+  },
+  Object {
+    "data": Object {
+      "URI": "https://mozilla.org",
+      "endTime": 8,
+      "fetchStart": 7.5,
+      "id": 7,
+      "pri": 0,
+      "startTime": 7,
+      "status": "STATUS_STOP",
+      "type": "Network",
+    },
+    "dur": 1,
+    "name": "Load 7: https://mozilla.org",
+    "start": 7,
+    "title": null,
+  },
+]
+`;
+
 exports[`snapshots of selectors/profile matches the last stored run of markerThreadSelectors.getMarkerChartTiming 1`] = `
 Array [
   Object {
     "end": Array [
       3,
     ],
     "index": Array [
-      0,
+      3,
```

Comment:
These changes are expected because the index now refers to the full marker list, not the search filtered list.